test/forward_merger.test.ts

```diff
@@ -0,0 +1,300 @@
+    forwardMerger.getBranches = mockGetBranches;
+    await forwardMerger.mergeForward();
+
+    expect(mockCreatePR.mock.calls[0][0].base).toBe("branch-22.04");
+  });
+
+  test("getBranches should return versioned branches", async () => {
+    const context = makePushContext({
+      ref: "refs/heads/branch-21.12",
+    });
+    const forwardMerger = new ForwardMerger(context, context.payload);
+    const branches = [
+      {
+        name: "branch-21.12",
+      },
+      {
+        name: "non-versioned",
+      },
+      {
+        name: "branch-21.10",
+      },
+    ];
+    mockPaginate.mockResolvedValue(branches);
+    mockListBranches.mockResolvedValue("something");
+    forwardMerger.context.octokit.repos.listBranches = mockListBranches as any;
+
+    const result = await forwardMerger.getBranches();
+
+    expect(result.length).toEqual(2);
+    expect(result[0]).toEqual("branch-21.12");
+    expect(result[1]).toEqual("branch-21.10");
+    expect(mockPaginate.mock.calls[0][0]).toBe(mockListBranches);
+    expect(mockPaginate.mock.calls[0][1]).toMatchObject({
+      owner: context.payload.repository.owner.login,
+      repo: context.payload.repository.name,
+    });
+  });
+
+  test("sortBranches should sort branches", async () => {
+    const context = makePushContext({
+      ref: "refs/heads/branch-21.12",
+    });
+    const forwardMerger = new ForwardMerger(context, context.payload);
+    const branches = [
+      "branch-0.10",
+      "branch-21.12",
+      "branch-21.10",
+      "branch-19.10",
+      "branch-22.02",
+      "branch-0.9",
+    ];
+
+    const result = forwardMerger.sortBranches(branches);
+
+    expect(result.length).toEqual(6);
+    expect(result[0]).toEqual("branch-0.9");
+    expect(result[1]).toEqual("branch-0.10");
+    expect(result[2]).toEqual("branch-19.10");
+    expect(result[3]).toEqual("branch-21.10");
+    expect(result[4]).toEqual("branch-21.12");
+    expect(result[5]).toEqual("branch-22.02");
+  });
+
+  test("getNextBranch should return next branch", async () => {
+    const context = makePushContext({
+      ref: "refs/heads/branch-21.12",
+    });
+    const forwardMerger = new ForwardMerger(context, context.payload);
+    const branches = ["branch-21.12", "branch-21.10", "branch-22.02"];
+    const sortedBranches = forwardMerger.sortBranches(branches);
+    const result = await forwardMerger.getNextBranch(sortedBranches);
+
+    expect(result).toEqual("branch-22.02");
+  });
+
+  test("getNextBranch should return null if there is no next branch", async () => {
+    const context = makePushContext({
+      ref: "refs/heads/branch-22.02",
+    });
+    const forwardMerger = new ForwardMerger(context, context.payload);
+    const branches = ["branch-21.12", "branch-21.10", "branch-22.02"];
+    const sortedBranches = forwardMerger.sortBranches(branches);
+    const result = await forwardMerger.getNextBranch(sortedBranches);
+
+    expect(result).toBeFalsy();
```

Comment:
```suggestion
    expect(result).toBe(undefined);
```


let's be explicit here to match the return type. there are many falsy values.